docs/advanced.rst

```diff
@@ -1,3 +1,7 @@
+==============
```

Comment:
I have a PR underway to rework the doc, this file is currently not included in any toctree and sphinx complains with:
```
WARNING: document isn't included in any toctree
```